code/game/machinery/computer/abnormality_queue.dm

```diff
@@ -96,6 +96,8 @@
 	if(lock_after)
 		message_admins("[usr] has [logstring] the anomaly to [initial(target_type.name)].")
 		locked = TRUE
+		// PE awarded for yellow roll - just as kirie had wanted.
```

Comment:
Add a little chat message for this, something like "Headquarters has reimbursed you the cost of extracting a specific abnormality"